### ISSUE TITLE
Explorer search removes all standards

### ISSUE THREAD

**Issue by robvanderveer** [OWASP/OpenCRE#544]
The amazing new native explorer looks great. I observed that when you enter a search term such as session, that the linked standards are no longer displayed. I doubt this is by design. Can it be fixed?

**Comment by dkrizhanovskyi** [comment]
Hi Team,  

I came across this issue and would like to contribute. My background includes extensive experience in cryptographic protocols, post-quantum cryptography, and secure distributed systems, which I believe aligns well with the goals of OpenCRE.  

Here’s how I propose to approach this task:  
1. **Understanding the Problem**: I will thoroughly review the relevant sections of the codebase or CRE mappings related to this issue.  
2. **Developing a Solution**: Depending on the issue's nature, I can assist by:
   - Mapping new security standards or improving existing mappings using the provided CRE templates.
   - Enhancing code or processes, especially if related to automating mappings or ensuring consistency across CREs.
3. **Testing and Documentation**: I’ll ensure that my contributions meet project quality standards and include clear documentation for maintainers and future contributors.

Before proceeding, I’d like to confirm:
- Any specific details or constraints I should keep in mind?  
- Are there additional resources or examples that would help me better understand the context of this issue?

Looking forward to your guidance!  

Best regards,  
Daniil Krizhanovskyi  
(*GitHub handle: dkrizhanovskyi*)

**Comment by dkrizhanovskyi** [comment]
**Subject**: Proposal to Contribute  

Hi Team,  

I’d like to contribute to this issue. Could you clarify the exact requirements? Specifically:  
- Is this focused on updating mappings, automating linking, or a different enhancement?  
- Are there any guidelines or resources I should review before starting?  

Once clarified, I can begin work immediately. Let me know how I can assist!  

Best regards,  
Daniil Krizhanovskyi  
(*GitHub: dkrizhanovskyi*)

**Comment by robvanderveer** [comment]
Thanks Daniil. We have no further information for you, other than that the display of mapped standards stop once you start a search. Looking forward to your work!
to @dkrizhanovskyi

**Comment by skn8733** [comment]
Hi @robvanderveer ! I wanted to check and see if this issue is still on going?

**Comment by robvanderveer** [comment]
yes it's still a problem. It is about https://www.opencre.org/explorer
